terraform/gcp/projects/meom-ige.tfvars

```diff
@@ -4,7 +4,7 @@ project_id = "meom-ige-cnrs"
 zone   = "us-central1-b"
 region = "us-central1"
 
-core_node_machine_type = "n1-highmem-2"
+core_node_machine_type = "n2-highmem-4"
```

Comment:
Is the doubling of this necessary? They don't have as many pods as the shared cluster does.

Reply:
I'd say yes, as a daskhub cluster this should be a n2-highmem-4 in order to fit a large prometheus server. I'll get them to run only one such node!